modin/core/io/pickle/pickle_dispatcher.py

```diff
@@ -47,10 +47,10 @@ def _read(cls, filepath_or_buffer, **kwargs):
         The number of partitions is equal to the number of input files.
         """
         if not (isinstance(filepath_or_buffer, str) and "*" in filepath_or_buffer):
-            warnings.warn("Defaulting to Modin core implementation")
             return cls.single_worker_read(
                 filepath_or_buffer,
                 single_worker_read=True,
+                reason="Buffers and wildcard file names are not supported",
```

Comment:
This is checking if a wildcard is in the buffer's path right? I think we should change the error message to something like
```suggestion
                reason="Buffers with wildcard file paths are not supported",
```

Reply:
Actually, it seems like I misunderstood the condition for this branch: `not (isinstance(filepath_or_buffer, str) and "*" in filepath_or_buffer))` will hold if (1) the argument is not a string (i.e. is a buffer), or (2) the argument is a string and does NOT contain a wildcard. If my interpretation is correct, then this error message should be to the effect of ""Buffers or filenames without wildcards are not supported."